ui-admin/src/study/participants/survey/SurveyResponseView.tsx

```diff
@@ -100,7 +100,10 @@ export function RawEnrolleeSurveyView({
               <DropdownButton
                 onClick={() => setIsEditing(true)}
                 icon={faPencil}
-                disabled={!configSurvey.survey.allowAdminEdit}
+                disabled={
+                  !configSurvey.survey.allowAdminEdit ||
+                    !userHasPermission(user, studyEnvContext.portal.id, 'survey_response_edit')
+                }
```

Comment:
ux nit: should there be a tooltip explaining that you do not have permissions? i could see being confused when you can't do something that you can see. or just not render the dropdown to edit.

Reply:
that's a good point- I kinda went back and forth on whether or not to display it. I don't think we're consistent about this in our UI, but I wonder if we want to establish some patterns like:

* If the user can't perform the action and there's nothing they can do on their own to enable the action, don't display it at all (i.e. they don't have the right permission or role to do the action)
* If the user can't perform the action but they can do something to enable it, display it with a tooltip telling them what they can do. An example would be fixing an error in a survey question before being able to click the 'Save' button

So in this case, I wouldn't render the option here at all.

I do think there are cases where the user might want to know that the action is possible so they can ask someone on their team to grant them that action, but I'm not sure how important/useful that is

Reply:
👍 I'll add a ticket for this